nitransforms/linear.py

```diff
@@ -326,6 +342,13 @@ def load(filename, fmt='X5', reference=None):
     # elif fmt.lower() == 'afni':
     #     parameters = LPS.dot(self.matrix.dot(LPS))
     #     parameters = parameters[:3, :].reshape(-1).tolist()
+    elif fmt.lower() in ('fs', 'lta'):
+        with open(filename) as ltafile:
+            lta = LinearTransformArray.from_fileobj(ltafile)
+        assert lta['nxforms'] == 1  # ever have multiple transforms?
```

Comment:
Yes, we should be able to have multiple transforms (i.e., nxforms does not need to be 1)